Error adding tenant with trusted-ca in Hono deployed via Helm

Adding a tenant with a trusted-ca results in a 400 Bad Request response if Hono has been deployed using the standard helm chart.

Error in device registry log:
````
20:01:22.866 [vert.x-eventloop-thread-0] DEBUG o.e.h.d.FileBasedTenantService - error parsing payload of add tenant request
java.lang.IllegalArgumentException: improperly specified input name: ${trusted-ca.subject-dn}
	at java.base/javax.security.auth.x500.X500Principal.<init>(X500Principal.java:183)
	at java.base/javax.security.auth.x500.X500Principal.<init>(X500Principal.java:128)
	at org.eclipse.hono.util.TenantObject.lambda$getTrustedCaSubjectDn$0(TenantObject.java:130)
	at java.base/java.util.Optional.map(Optional.java:265)
	at org.eclipse.hono.util.TenantObject.getTrustedCaSubjectDn(TenantObject.java:130)
	at org.eclipse.hono.deviceregistry.FileBasedTenantService.lambda$getByCa$10(FileBasedTenantService.java:392)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176)
	at java.base/java.util.HashMap$ValueSpliterator.tryAdvance(HashMap.java:1696)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
	at org.eclipse.hono.deviceregistry.FileBasedTenantService.getByCa(FileBasedTenantService.java:393)
	at org.eclipse.hono.deviceregistry.FileBasedTenantService.add(FileBasedTenantService.java:314)
	at org.eclipse.hono.deviceregistry.FileBasedTenantService.add(FileBasedTenantService.java:291)
	at org.eclipse.hono.service.tenant.CompleteBaseTenantService.processAddRequest(CompleteBaseTenantService.java:93)
	at org.eclipse.hono.service.tenant.CompleteBaseTenantService.processRequest(CompleteBaseTenantService.java:67)
	at org.eclipse.hono.service.EventBusService.processRequestMessage(EventBusService.java:160)
	at io.vertx.core.eventbus.impl.HandlerRegistration.deliver(HandlerRegistration.java:264)
	at io.vertx.core.eventbus.impl.HandlerRegistration.handle(HandlerRegistration.java:242)
	at io.vertx.core.eventbus.impl.EventBusImpl$InboundDeliveryContext.next(EventBusImpl.java:565)
	at io.vertx.core.eventbus.impl.EventBusImpl.lambda$deliverToHandler$5(EventBusImpl.java:524)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
	at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:462)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:897)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.io.IOException: Incorrect AVA format
	at java.base/sun.security.x509.AVA.readChar(AVA.java:555)
	at java.base/sun.security.x509.AVA.<init>(AVA.java:181)
	at java.base/sun.security.x509.AVA.<init>(AVA.java:147)
	at java.base/sun.security.x509.RDN.<init>(RDN.java:147)
	at java.base/sun.security.x509.X500Name.parseDN(X500Name.java:928)
	at java.base/sun.security.x509.X500Name.<init>(X500Name.java:164)
	at java.base/javax.security.auth.x500.X500Principal.<init>(X500Principal.java:181)
	... 32 common frames omitted
````

--- a/deploy/pom.xml
+++ b/deploy/pom.xml
@@ -230,6 +230,8 @@
                             </resources>
                             <filters>
                                 <filter>${project.basedir}/src/main/kubernetes-config.properties</filter>
+                                <filter>${project.build.directory}/config/hono-demo-certs-jar/trust-anchor.properties</filter>
+                                <filter>${project.build.directory}/config/hono-demo-certs-jar/device-certs.properties</filter>
                             </filters>
                         </configuration>
                     </execution>

